Arch IFC: Exporting rectangular section to ifc

marvellouschandan opened the pull request:
Thank you for creating a pull request to contribute to FreeCAD! To ease integration, please confirm the following:

- [ ] Branch rebased on latest master `git pull --rebase upstream master`
- [ ] Unit tests confirmed to pass by running `./bin/FreeCAD --run-test 0`
- [ ] Commit message is [well-written](https://chris.beams.io/posts/git-commit/)
- [ ] Commit message includes `issue #<id>` or `fixes #<id>` where `<id>` is the [associated MantisBT](https://freecadweb.org/wiki/tracker#GitHub_and_MantisBT) issue id if one exists

And please remember to update the Wiki with the features added or changed once this PR is merged.

---

marvellouschandan commented:
https://forum.freecadweb.org/viewtopic.php?f=39&t=31965&e=1&view=unread#unread

Follow this link for the thread regarding this pull request.

amrit3701 commented:
@marvellouschandan Welcome for the contribution!

In your last commit (https://github.com/FreeCAD/FreeCAD/pull/1766/commits/ec0e8b510aaaf349ab30956cd0afcb36797aa296) there is something wrong as git show you delete and paste all content of importIFC.py file.

Also, all your three commits belonging to one issue and I suggest to first merge them into one and then send a new PR.

Thanks

marvellouschandan commented:
> @marvellouschandan Welcome for the contribution!

Thanks @amrit3701 

> 
> In your last commit ([ec0e8b5](https://github.com/FreeCAD/FreeCAD/commit/ec0e8b510aaaf349ab30956cd0afcb36797aa296)) there is something wrong as git show you delete and paste all content of importIFC.py file.

Actually I did a mistake, I changed the importIFC.py file in root installation directory of FreeCAD, when I found that the solution is working fine. I replaced the changed file with the file in the git folder of mine and then pushed into my master. This was the issue.

> 
> Also, all your three commits belonging to one issue and I suggest to first merge them into one and then send a new PR.

Ok will inform after trying this

> 
> Thanks
